SampleApp/Views/Reusable/LoadingImage.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  LoadingImage.swift
+//  SampleApp
+//
+//  Created by Benjamin Patch on 9/29/24.
+//
+
+import SwiftUI
+
+struct LoadingImage: View {
+    var body: some View {
+        ZStack(alignment: .center) {
+            ProgressView()
+                .progressViewStyle(.circular)
+                .controlSize(.extraLarge)
+                .tint(Color.yellow)
+                .padding(.bottom, 30)
+            Image(systemName: "mountain.2.fill")
+                .resizable()
+                .aspectRatio(contentMode: .fit)
+                .frame(maxHeight: 50)
+        }
+        .aspectRatio(1, contentMode: .fit)
+        .background(Gradient(colors: [.blue, .blue, .clear, .clear, .clear]), in: RoundedRectangle(cornerSize: .init(width: 20, height: 20)))
+        .frame(maxWidth: .infinity)
+    }
+}
+
+#Preview {
+    LoadingImage()
```

Comment:
This guy was kind of a fun little distraction. I wanted to make the loading spinner be like a sun behind the "default image" image.
<img width="70" alt="Screenshot 2024-09-29 at 11 46 31 PM" src="https://github.com/user-attachments/assets/e2102d43-aa16-4ef5-b88b-6c5ea6651441">